{"uid":"7d1a7676093c3954","name":"Rename an existing workspace test","fullName":"tests.workspaces_test.TestWorkspaces#test_rename_workspace","historyId":"cd63adf5d11191c5904362b968c9e131","time":{"start":1717204780693,"stop":1717204780693,"duration":0},"description":"Rename an existing workspace","descriptionHtml":"<p>Rename an existing workspace</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e018500e8a <unknown>\n#1 0x55e0181ea45c <unknown>\n#2 0x55e0182365b5 <unknown>\n#3 0x55e018236671 <unknown>\n#4 0x55e01827af14 <unknown>\n#5 0x55e0182594dd <unknown>\n#6 0x55e0182782cc <unknown>\n#7 0x55e018259253 <unknown>\n#8 0x55e0182291c7 <unknown>\n#9 0x55e018229b3e <unknown>\n#10 0x55e0184c72cb <unknown>\n#11 0x55e0184cb377 <unknown>\n#12 0x55e0184b3dfe <unknown>\n#13 0x55e0184cbe42 <unknown>\n#14 0x55e01849879f <unknown>\n#15 0x55e0184f0178 <unknown>\n#16 0x55e0184f034b <unknown>\n#17 0x55e0184fffbc <unknown>\n#18 0x7f431a094ac3 <unknown>","statusTrace":"self = <tests.workspaces_test.TestWorkspaces object at 0x7f88c786e8d0>\n\n    @pytest.fixture(autouse=True)\n    def setup_method_fixture(self):\n>       login(self.about_page, self.login_page)\n\ntests/workspaces_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/workspaces_test.py:15: in login\n    about_page.click_login_link()\npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"2d0887301a7523cb25f0ef3fe66a8e96\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f88c6f8cf40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55e018500e8a <unknown>\nE       #1 0x55e0181ea45c <unknown>\nE       #2 0x55e0182365b5 <unknown>\nE       #3 0x55e018236671 <unknown>\nE       #4 0x55e01827af14 <unknown>\nE       #5 0x55e0182594dd <unknown>\nE       #6 0x55e0182782cc <unknown>\nE       #7 0x55e018259253 <unknown>\nE       #8 0x55e0182291c7 <unknown>\nE       #9 0x55e018229b3e <unknown>\nE       #10 0x55e0184c72cb <unknown>\nE       #11 0x55e0184cb377 <unknown>\nE       #12 0x55e0184b3dfe <unknown>\nE       #13 0x55e0184cbe42 <unknown>\nE       #14 0x55e01849879f <unknown>\nE       #15 0x55e0184f0178 <unknown>\nE       #16 0x55e0184f034b <unknown>\nE       #17 0x55e0184fffbc <unknown>\nE       #18 0x7f431a094ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup_method_fixture","time":{"start":1717204782337,"stop":1717204792676,"duration":10339},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e018500e8a <unknown>\n#1 0x55e0181ea45c <unknown>\n#2 0x55e0182365b5 <unknown>\n#3 0x55e018236671 <unknown>\n#4 0x55e01827af14 <unknown>\n#5 0x55e0182594dd <unknown>\n#6 0x55e0182782cc <unknown>\n#7 0x55e018259253 <unknown>\n#8 0x55e0182291c7 <unknown>\n#9 0x55e018229b3e <unknown>\n#10 0x55e0184c72cb <unknown>\n#11 0x55e0184cb377 <unknown>\n#12 0x55e0184b3dfe <unknown>\n#13 0x55e0184cbe42 <unknown>\n#14 0x55e01849879f <unknown>\n#15 0x55e0184f0178 <unknown>\n#16 0x55e0184f034b <unknown>\n#17 0x55e0184fffbc <unknown>\n#18 0x7f431a094ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 884, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 25, in setup_method_fixture\n    login(self.about_page, self.login_page)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/tests/workspaces_test.py\", line 15, in login\n    about_page.click_login_link()\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Click Login link","time":{"start":1717204782337,"stop":1717204792676,"duration":10339},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e018500e8a <unknown>\n#1 0x55e0181ea45c <unknown>\n#2 0x55e0182365b5 <unknown>\n#3 0x55e018236671 <unknown>\n#4 0x55e01827af14 <unknown>\n#5 0x55e0182594dd <unknown>\n#6 0x55e0182782cc <unknown>\n#7 0x55e018259253 <unknown>\n#8 0x55e0182291c7 <unknown>\n#9 0x55e018229b3e <unknown>\n#10 0x55e0184c72cb <unknown>\n#11 0x55e0184cb377 <unknown>\n#12 0x55e0184b3dfe <unknown>\n#13 0x55e0184cbe42 <unknown>\n#14 0x55e01849879f <unknown>\n#15 0x55e0184f0178 <unknown>\n#16 0x55e0184f034b <unknown>\n#17 0x55e0184fffbc <unknown>\n#18 0x7f431a094ac3 <unknown>\n\n","statusTrace":"  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/about_page.py\", line 20, in click_login_link\n    self.click(self.LOGIN_LINK)\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/pages/base_page.py\", line 60, in click\n    el: WebElement = self.wait.until(\n                     ^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/Selenium-Python-Example/Selenium-Python-Example/.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"testStage":{"description":"Rename an existing workspace","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55e018500e8a <unknown>\n#1 0x55e0181ea45c <unknown>\n#2 0x55e0182365b5 <unknown>\n#3 0x55e018236671 <unknown>\n#4 0x55e01827af14 <unknown>\n#5 0x55e0182594dd <unknown>\n#6 0x55e0182782cc <unknown>\n#7 0x55e018259253 <unknown>\n#8 0x55e0182291c7 <unknown>\n#9 0x55e018229b3e <unknown>\n#10 0x55e0184c72cb <unknown>\n#11 0x55e0184cb377 <unknown>\n#12 0x55e0184b3dfe <unknown>\n#13 0x55e0184cbe42 <unknown>\n#14 0x55e01849879f <unknown>\n#15 0x55e0184f0178 <unknown>\n#16 0x55e0184f034b <unknown>\n#17 0x55e0184fffbc <unknown>\n#18 0x7f431a094ac3 <unknown>","statusTrace":"self = <tests.workspaces_test.TestWorkspaces object at 0x7f88c786e8d0>\n\n    @pytest.fixture(autouse=True)\n    def setup_method_fixture(self):\n>       login(self.about_page, self.login_page)\n\ntests/workspaces_test.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \ntests/workspaces_test.py:15: in login\n    about_page.click_login_link()\npages/about_page.py:20: in click_login_link\n    self.click(self.LOGIN_LINK)\npages/base_page.py:60: in click\n    el: WebElement = self.wait.until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"2d0887301a7523cb25f0ef3fe66a8e96\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7f88c6f8cf40>\nmessage = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x55e018500e8a <unknown>\nE       #1 0x55e0181ea45c <unknown>\nE       #2 0x55e0182365b5 <unknown>\nE       #3 0x55e018236671 <unknown>\nE       #4 0x55e01827af14 <unknown>\nE       #5 0x55e0182594dd <unknown>\nE       #6 0x55e0182782cc <unknown>\nE       #7 0x55e018259253 <unknown>\nE       #8 0x55e0182291c7 <unknown>\nE       #9 0x55e018229b3e <unknown>\nE       #10 0x55e0184c72cb <unknown>\nE       #11 0x55e0184cb377 <unknown>\nE       #12 0x55e0184b3dfe <unknown>\nE       #13 0x55e0184cbe42 <unknown>\nE       #14 0x55e01849879f <unknown>\nE       #15 0x55e0184f0178 <unknown>\nE       #16 0x55e0184f034b <unknown>\nE       #17 0x55e0184fffbc <unknown>\nE       #18 0x7f431a094ac3 <unknown>\n\n.venv/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","steps":[{"name":"public ip address","time":{"start":1717204792689,"stop":1717204792718,"duration":29},"status":"passed","steps":[],"attachments":[{"uid":"130c192f9f43ed1d","name":"HTTP logs of http://checkip.amazonaws.com/","source":"130c192f9f43ed1d.txt","type":"text/plain","size":357}],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":1,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"7b7f7870c1344554","name":"Cookies","source":"7b7f7870c1344554.json","type":"application/json","size":3637},{"uid":"a4010a53437f7320","name":"Session Storage","source":"a4010a53437f7320.json","type":"application/json","size":359},{"uid":"f37382c9be35d50","name":"Local Storage","source":"f37382c9be35d50.json","type":"application/json","size":294},{"uid":"a187c543e52b230c","name":"Console Logs","source":"a187c543e52b230c.json","type":"application/json","size":6729},{"uid":"191517e506bebf82","name":"Full Page Screenshot","source":"191517e506bebf82.png","type":"image/png","size":5696115},{"uid":"5397bb8d945f7a2e","name":"URL","source":"5397bb8d945f7a2e.uri","type":"text/uri-list","size":23},{"uid":"6dc1853e308f6251","name":"Network Logs","source":"6dc1853e308f6251.json","type":"application/json","size":942956}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":1,"attachmentsCount":8,"hasContent":true,"attachmentStep":false},"afterStages":[],"labels":[{"name":"story","value":"WorkSpaces Creation and Editing Functionality"},{"name":"severity","value":"normal"},{"name":"epic","value":"Workspaces"},{"name":"tag","value":"run(order=2)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"workspaces_test"},{"name":"subSuite","value":"TestWorkspaces"},{"name":"host","value":"fv-az985-174"},{"name":"thread","value":"2328-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.workspaces_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["run(order=2)"]},"source":"7d1a7676093c3954.json","parameterValues":[]}